docs/workflows/transitions/preprocess/preprocess/make_predictions.py

```diff
@@ -128,7 +128,8 @@ def make_predictions(las_client, event):
                 predictions = concatenate_lines_from_different_pages(predictions, field_config)
 
             predictions = patch_empty_predictions(predictions, labels, no_empty_prediction_fields)
-            predictions = filter_away_low_confidence_lines(predictions, field_config)
+            filter_lines_config = model_metadata.get('Cradl:filterLowLinesConfig', {})
```

Comment:
```suggestion
            filter_lines_config = model_metadata.get('cradl:filterLowLinesConfig', {})
```

Reply:
From a user perspective it seems like this should be part of the predictionConfig, doesn't it? 
what if we had something like?
```
cradl:predictionConfig: 
    filter_prediction_settings:
         average_line_confidence_threshold: ...
         include_all_columns: ...
```

Any thoughts on this one @augustak